cmd/tendermint/commands/root.go

```diff
@@ -26,7 +27,11 @@ var RootCmd = &cobra.Command{
 		err := viper.Unmarshal(config)
 		config.SetRoot(config.RootDir)
 		cfg.EnsureRoot(config.RootDir)
-		logger = log.NewFilterByLevel(logger, config.LogLevel)
+		if config.LogLevel == "info" || config.LogLevel == "error" || config.LogLevel == "debug" {
+			logger = log.NewFilterByLevel(logger, config.LogLevel)
```

Comment:
Would it be bad to have this logic in tmlibs too? Maybe NewFilterByLevel returns an error, or we check for nil logger?